splash/browser_tab.py

```diff
@@ -104,6 +106,11 @@ def _init_webpage(self, verbosity, network_manager, splash_proxy_factory, render
         self.web_view.resize(
             QSize(*map(int, defaults.VIEWPORT_SIZE.split('x'))))
 
+    def _init_elements_storage(self):
```

Comment:
It'd be nice to have this work lazily, when a first HtmlElement instance is created. This way Splash won't be changing global scope unless required.